taxi/form.py

```diff
@@ -0,0 +1,51 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.core.validators import RegexValidator
+
+from taxi.models import Driver, Car
+
+
+class DriverCreateForm(UserCreationForm):
+    class Meta:
+        model = Driver
+        fields = [
+            "username",
+            "first_name",
+            "last_name",
+            "license_number",
+        ]
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    license_number = forms.CharField(
+        required=True,
+        validators=[
+            RegexValidator(
+                regex=r"^[A-Z]{3}\d{5}$",
+                message="Number of license must consist "
+                        "only of 8 characters, first 3 characters "
+                        "are uppercase letters, "
+                        "last 5 characters are digits."
```

Comment:
The regex pattern `r"^[A-Z]{3}\d{5}$"` requires exactly 3 uppercase letters followed by 5 digits, which matches the message description. However, ensure that this pattern aligns with the actual requirements for a license number. If the requirements differ, adjust the regex pattern or the message accordingly.